internal/store/pod.go

```diff
@@ -1022,6 +1022,22 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_pod_host_network",
+			Type: metric.Gauge,
+			Help: "Describes if pod have host network.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				return &metric.Family{
+					Metrics: []*metric.Metric{
+						{
+							LabelKeys:   []string{},
+							LabelValues: []string{},
+							Value:       boolFloat64(p.Spec.HostNetwork),
```

Comment:
why not put this into the `kube_pod_info` metric?

Reply:
yeah, first commit was about adding additional label to kube_pod_info.
If this better, I can revert commits

Reply:
yeah I think that's better

Reply:
Ok, I just revert commits
But if you want, you can squash on merge